style.css

```diff
@@ -0,0 +1,51 @@
+* {
+  box-sizing: border-box;
+}
+
+@media screen and (max-width: 480px) {
+  main {
+    display: flex;
+    margin: auto;
+  }
+
+  header {
+    display: flex;
+    position: absolute;
+    color: hsl(234, 100%, 69%);
+    width: 80%;
+    margin: 20px;
+  }
+
+  .logo {
+    font-size: 25px;
+  }
+
+  .hamburger {
+    color: #172bdb;
+    width: 55px;
+    height: 90px;
```

Comment:
Please ensure you must stick to the design as much as possible (e.g. font, colors, images, margins, social icons) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

### Your design:
![your-design](https://user-images.githubusercontent.com/10332499/148507488-fe3ead07-d2c2-48df-ae8b-b05dd285878a.PNG)

### Figma design:
![figma](https://user-images.githubusercontent.com/10332499/148507524-12c870a4-18e9-4937-b4a4-56134dc0ff6c.PNG)